docs/vgclog.md

```diff
@@ -0,0 +1,423 @@
+
+- Tuning GC performance to reduce pause times and speed up GC operations
+
+- Troubleshooting GC operations and policies. For example, analyzing long pauses, or determining how free memory is divided in the heap before and after a GC cycle. 
+
+Verbose GC logs, when enabled, begin capturing information as soon as garbage collection is initialized. By default, the information is printed to STDERR. The information can also be [printed to a file](vgclog.md#how-to-generate-a-verbonse-garbage-collection-log).
+
+To help you visualize and analyze the garbage collection, you can feed verbose GC log files into various diagnostic tools and interfaces. Examples include tools such as the [Garbage Collection and memory visualizer extension for Eclipse](https://marketplace.eclipse.org/content/ibm-monitoring-and-diagnostic-tools-garbage-collection-and-memory-visualizer-gcmv), and online services such as [GCEasy](https://gceasy.io). 
+
+**Note:** You can run one or more GC traces by using the (['-XTgc' option](xtgc.md)) to get more detailed information to help diagnose GC problems or perform finer tuning.
+
+## How to generate a verbose garbage collection log 
+
+You can enable verbose GC logs by specifying the `-verbose:gc` standard option when you start your application. For more information, see [standard command-line options](cmdline_general.md).
+
+The output of `-verbose:gc` is printed to STDERR by default. To print the log output to a file, append the `-Xverbosegclog` option. You can print the verbose GC log file to a succession of files, where each file logs a specified number of GC cycles, by using additional parameters of `-Xverbosegclog` option. For more information, see the command-line option [-Xverbosegclog](xverbosegclog.md).
+
+## Verbose GC log contents 
+
+The Verbose garbage collection logs are printed in XML format and consist of the following sections:  
+
+- a summary of your GC configuration, which is captured in the stanza beginning with the [`<INITIATIZED>` tag](vgclog.md#xml-tags-an-attributes) 
+
+- stanzas that contain information about the different garbage collection cycles run, including the GC increments and GC operations that made up the GC cycle. 
+
+The verbose GC logs are event-based, recording each *stop-the-world* event as it happens. While *stop-the-world* cycles are fully logged, some aspects of concurrent cycles are also logged, even though operations that are running concurrently with application threads are not logged. Concurrent operations run as part of a concurrent cycle, and these concurrent cycles begin and finish with *stop-the-world* events - events that are logged. (For some garbage collectors, the concurrent cycle also includes intermediate *stop-the-world* events). In addition, for some types of concurrent collections, verbose GC logs also include information about when certain targets are reached. 
```

Comment:
I'm a little confused by this paragraph as to the extent of what is recorded. Perhaps we can simplify this a little? Is this summary correct @amicic ? (for my understanding)

The following event-based information is recorded: 

- Global GC cycles: 
    - All *stop-the-world* (STW) events.
- Concurrent GC cycles: 
    - All STW events, which occur at the start and end of a cycle. For some garbage collectors, intermediate STW events are also recorded.
  - Information about when certain targets are reached. 

Operations that are running concurrently with application threads are not recorded.

Reply:
This is not quite right - see comment below. I really like the way this is structured, much clearer - I'll work to put that information in a clearer fashion.

Reply:
Part of the new draft